src/go/rdctl/pkg/runner/queue_test.go

```diff
@@ -0,0 +1,97 @@
+package runner
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+)
+
+func TestTaskRunner(t *testing.T) {
+	t.Run("should run all functions if context not cancelled and no errors", func(t *testing.T) {
+		ctx := context.Background()
+		taskRunner := NewTaskRunner(ctx)
+		ranSlice := []bool{false, false, false}
+		for i := range ranSlice {
+			i := i
+			taskRunner.Add(func() error {
+				ranSlice[i] = true
+				return nil
+			})
+		}
+		if err := taskRunner.Wait(); err != nil {
```

Comment:
We often use `github.com/stretchr/testify` (but I guess there's no obligation to do so).

Reply:
I modified the `TaskRunner` unit tests to use `testify`. I left the new `Manager` unit tests as they are so they remain consistent with the existing `Manager` unit tests.